netty/src/main/java/io/grpc/netty/NettyClientTransport.java

```diff
@@ -110,6 +112,8 @@
     this.maxHeaderListSize = maxHeaderListSize;
     this.authority = new AsciiString(authority);
     this.userAgent = new AsciiString(GrpcUtil.getGrpcUserAgent("netty", userAgent));
+    // TODO(lukaszx0): do we want to reuse, or create new key for server/client?
+    this.attributes = Attributes.newBuilder().set(Grpc.TRANSPORT_ATTR_REMOTE_ADDR, address).build();
```

Comment:
Do we need to make simpler change to other classes inheriting from `ConnectionClientTransport` (e.g., `InProcessTransport`)?